Material_FAB/index.ets

```diff
@@ -1,3 +1,17 @@
-export { rippleUtils } from './src/main/ets/components/MainPage/ripple'
```

Comment:
Change the lib folder name from Material_FAB to material-fab

Reply:
library name can't have '-' in it